bitrise.yml

```diff
@@ -148,7 +148,7 @@ workflows:
     meta:
       bitrise.io:
         stack: osx-xcode-14.3.x-ventura
-        machine_type_id: g2.8core
+        machine_type_id: g2-m1.4core
```

Comment:
This is not a valid stack on Bitrise. When you try to trigger the build there is this message:
`Failed to approve build request: response code: 400, message: Failed to start build: Invalid machine type for workflow: 'configure_build_fennec'.`
The correct one should be: `g2-m1.8core`